### ISSUE TITLE
Ktlint-cli `--code-style=intellij_idea` is not respected in ktlint `1.0.0`

### ISSUE THREAD

**Issue by voegelas** [pinterest/ktlint#2238]
## Expected Behavior
In the following example, ktlint should not complain about "multiline-expression-wrapping" if the code style is set to "intellij_idea".

```kotlin
val chars = "foo".map { c ->
    c
}
```

## Observed Behavior
ktlint 1.0.0 outputs a "multiline-expression-wrapping" warning and inserts a newline and an extra indentation level if the code is formatted. Intellij IDEA doesn't do that and puts the lambda expression back on the first line if the code is formatted twice. ktlint 0.50.0 works as expected.

```kotlin
val chars =
    "foo".map { c ->
        c
    }
```
## Steps to Reproduce
Run `ktlint --code-style=intellij_idea example.kts` on the first example.

## Your Environment
* Version of ktlint used: 1.0.0
* Operating System and version: Fedora 38 with OpenJDK 17 and Kotlin 1.9.0

**Comment by paul-dingemans** [comment]
Property `--code-style=intellij_idea` is indeed not passed correctly to ktlint rule engine. Also in previous versions this was not done correctly but in those releases the default codestyle was `intellij_idea`.

When you specify the code style in the `.editorconfig` file, it is respected though. This is also the preferred way to specify the code style as it will then be used by everyone working on the same project (provided that the `.editorconfig` is committed to the repository).
```
root = true

[*.{kt,kts}]
ktlint_code_style = intellij_idea
```

**Comment by voegelas** [comment]
I think that it's useful to be able to pass the code style on the command line. I've started to use Kotlin for scripting too. Adding an `.editorconfig` file to each and every script and repository is cumbersome. I've got the following settings in my Vim configuration:

```vim
let b:ale_fixers = ['ktlint']
let b:ale_kotlin_ktlint_options = join([
  \ '--log-level=none',
  \ '--code-style=intellij_idea',
  \ ])
```
